rafs/src/metadata/layout/v6.rs

```diff
@@ -1640,6 +1658,75 @@ impl RafsXAttrs {
     }
 }
 
+#[derive(Clone, Default, Debug)]
+pub struct RafsV6PrefetchTable {
+    /// List of inode numbers for prefetch.
+    /// Note: It's not inode index of inodes table being stored here.
+    pub inodes: Vec<u64>,
```

Comment:
Hi, we could use 32bit nid here, since blob.meta won't be too large for nydus v6 format.
it can support 4*32 = 128GB bootstrap size, which I think it's enough

Reply:
already changed to u32